terraform/modules/services/sat/ecs.tf

```diff
@@ -55,3 +55,63 @@ resource "aws_ecs_task_definition" "sat-task-definition" {
 
   # add volume? So we dont have to keep downloading same docker image
 }
+
+
+resource "aws_ecs_task_definition" "sat-clean-up-task-definition" {
+  family                   = "sat-clean-up"
+  requires_compatibilities = ["FARGATE"]
+  network_mode             = "awsvpc"
+
+  # specific values are needed -
+  # https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
+  cpu    = 1024
+  memory = 5120
```

Comment:
These can probably be a lot smaller, it just sends API requests, no data processing at all

Reply:
yea, thanks